cfn-guard/src/lib.rs

```diff
@@ -377,6 +377,136 @@ fn apply_rule_operation(
                 None
             }
         }
+        enums::OpCode::LessThan => {
+            match rule.rule_vtype {
+                enums::RValueType::Value | enums::RValueType::Variable => {
+                    if util::format_value(&val).parse::<f32>().unwrap() > util::strip_ws_nl(rule_val.to_string()).parse::<f32>().unwrap() ||
```

Comment:
Similar to my comment on GreaterThanOrEqaulTo, I would just use the rust `<` numeric operator here and flip the pass/fail case.